exercises/ocr-numbers/description.md

```diff
@@ -38,10 +38,10 @@ Update your program to recognize multi-character binary strings, replacing garbl
 Update your program to recognize all numbers 0 through 9, both individually and as part of a larger string.
 
 ```text
- _ 
+ _
  _|
-|_ 
-   
+|_
+
```

Comment:
Are you sure that it's not intended here?

Reply:
I believe that it's not intended.

And if I run `configlet generate` using a `problem-specifications` repo that's checked out to this PR then I get:
[before](https://github.com/ee7/python/blob/2d846bd69b3145e012c19e5e989d2a1516357c9b/exercises/ocr-numbers/README.md) vs [after](https://github.com/ee7/python/blob/7d1c76a267846cba6f886ee80d21050a6ac21593/exercises/ocr-numbers/README.md)

Reply:
Although we declared that the whitespace should be kept in https://github.com/exercism/problem-specifications/pull/1627#pullrequestreview-335650064, any decision is acceptable to me here, just wanted to make sure the previous information is considered.

Reply:
@petertseng Thanks for linking the previous discussion.

In my opinion: if this PR's whitespace changes to `ocr-numbers` are not significant then now is a good opportunity to make this change (now that there's no version string we have to bump).

But this PR includes a whitespace change that was said to be significant in that past discussion. Why is it significant? Pinging @NobbZ, who wrote:  
> The whitespace in this code block is significant and not allowed to be removed

I just want to be able to:
1. Run `configlet generate`.
1. Commit the `README.md` changes without seeing an error (I configure `git` to complain about trailing whitespace).

Reply:
It's significant in that exercise. It _must_ output a newline.

Add `<!-- prettier-ignore -->`, or better `<!-- prettier-ignore-start -->` `<!-- prettier-ignore-end-->` around the codeblock to solve this.

Reply:
Okay, thanks. I've reverted the change to `ocr-numbers`.

My motivation for this PR was fixing the whitespace in `resistor-color` and `resistor-color-duo`, rather than poking things elsewhere.